.github/workflows/circleci-trigger.yml

```diff
@@ -0,0 +1,89 @@
+name: CircleCI tests trigger
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches:
+      - main
+
+env:
+  PYTHONUNBUFFERED: 1
+  FORCE_COLOR: 1
+
+jobs:
+  circleci-trigger:
+    name: CircleCI tests trigger
+    environment: circleci_secrets
+    runs-on: ubuntu-latest
+    steps:
+      - name: Ping CircleCI pipeline
+        run: |
+          if [[ "${{ github.ref_type }}" == "tag" ]]; then
+            github_tag=${{ github.ref_name }}
+          else github_tag=""
+          fi
```

Comment:
Do we need this? Do we ever trigger this job from a tag rather than a branch?

Reply:
Yes, we have release tests for the new tags.

Reply:
Ah ok. That leads to more questions though, sorry... 😅 
1. How would these get triggered? I see there's a step `Tag main and create release on github` that creates the tag in our release workflow, but then there's nothing in the `on:` section of this file that will respond to it?
2. What do we actually want to run before a release? It seems a bit random to run just the QA tests but nothing else? I think we should either:
a. decide we want to test everything before release so add triggers into other GHA jobs
b. decide we don't want to test anything before release

Option b is fine for me since all the checks will run in the PR that merges to main anyway. But If we want to do option a then probably the best way is to modify our release job so that it runs all required jobs as `needs` before doing the release steps: https://stackoverflow.com/a/71489231.

Reply:
For the record, we are removing this check. It doesn't seem to test anything useful, especially given how the `hatch build` process works so that all files in `src/vizro` that are on git are always included.